wallet/wallet.go

```diff
@@ -489,7 +574,7 @@ func (w *Wallet) syncWithChain() error {
 			if err != nil {
 				return err
 			}
-			header, err := chainClient.GetBlockHeader(chainHash)
+			header, err := chainClient.GetBlockHeader(hash)
```

Comment:
what's this change for?

Reply:
dunno lol, good catch

Reply:
The prior diff is correct. This commit fixed a bug: https://github.com/Roasbeef/btcwallet/commit/257011e9f113aeea4720f03f732a19753a4599e1